iothub/service/src/ServicePointHelpers.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+
+namespace Microsoft.Azure.Devices
+{
+    // This type manages changing HttpClient defaults to more appropriate values
+    // There are two limits we target:
+    // - Per Server Connection Limit
+    // - Keep Alive Connection Timeout
+    // On .NET Core 2.1 & NET 5.0 the HttpClientTransport would default to using the SocketClientHandler
+    //   we adjust both limits on the client handler
+    // On .NET Standard & NET 4.6+ the HttpClientTransport would default to using the HttpClientHandler
+    //   and there is no easy way to set Keep Alive Connection Timeout but it's mitigated by WebHttpRequestTransport
+    //   being the default on NET 4.6.1
+    // The default transport on NET 4.6.1 is WebHttpRequestTransport the default and we are updating both
```

Comment:
the "...is WebHttpRequestTransport the default..." sounds weird. Missing a word?